Kitodo/src/main/java/org/kitodo/services/data/HistoryService.java

```diff
@@ -229,4 +238,9 @@ public void addAllObjectsToIndex() throws InterruptedException, IOException, Cus
         indexer.setMethod(HTTPMethods.PUT);
         indexer.performMultipleRequests(findAll(), historyType);
     }
+
+    @Override
+    public HistoryDTO convertJSONObjectToDTO(JSONObject jsonObject, boolean related) throws DataException {
+        return new HistoryDTO();
```

Comment:
So we will try to remember about it for future. In that moment it would add additional conflicts to rebase when next pr will come.